p4-fusion/branch_set.cc

```diff
@@ -207,15 +210,13 @@ std::unique_ptr<ChangedFileGroups> BranchSet::ParseAffectedFiles(const std::vect
 	branchIntegrationMap branchMap;
 	for (auto& clFileData : cl)
 	{
-		FileData fileData(clFileData);
-
 		// First, filter out files we don't want.
-		const std::string& depotFile = fileData.GetDepotFile();
+		const std::string& depotFile = clFileData.GetDepotFile();
```

Comment:
This local variable is only used in one place, might as well inline it at the usage site.